.env

```diff
@@ -27,3 +27,6 @@ DRUPAL_WEBROOT=public
 DRUPAL_SYNC_FILES=no
 
 OC_DEPLOYMENT_CONFIG_NAME=kasvatus-koulutus
+
+# URL for Elasticsearch
```

Comment:
Comment should be changed to:
`# URL for Elasticsearch, get correct port number with "make ps" and edit this variable if it is not 9200`

For the real version we should maybe think about enabling .env.local files, but probably depends on how often the port number changes.